Changed rainy observation check

mvyboh opened the pull request:
- changed the way rainy observations are checked so that all radars from last timestamp are checked (before user selected only 1)
- added support of rainbow files compressed to tar ball
- possible fix of memory leak issue of pyart module functions
- minor code cleaning and fixing

mvyboh committed:
Changed rainy observation check
- changed the way rainy observations are checked so that all radars from last timestamp are checked (before user selected only 1)
- minor code cleaning and fixing

mvyboh committed:
Dask num_workers quick fix

pavlikp commented on rainbow_to_h5.py:
```diff
@@ -104,68 +99,69 @@ def convert_rnbw_to_h5(conf, all_files, outfiles, datetimes_str, rain_thres):
+                    if product in ['dBZ', 'dBZv']:
+                        clmap = classify.filter_gabella(data, rm_nans=False, cartesian=True)
+                        data[np.nonzero(clmap)] = np.nan
+                        clmap = classify.filter_gabella(data, rm_nans=False, cartesian=True)
```
Toto je tu naschval 2-krat?

pavlikp replied:
to iste aj riadok 124

mvyboh replied:
fixnem v dalsom commite

mvyboh replied:
zmenene v https://github.com/kinit-sk/radar-dataset-preprocess/pull/1/commits/c27d9122555137ed3ce0fa99bedc5591e9ccae83

mvyboh committed:
Changed code to support tar as input
- tar files with rainbow files inside of them are now supported as input (previously user had to input rainbow files that were pre-extracted from tarballs)
- added gc around grid procedures from pyart module to solve memory leak issues

mvyboh committed:
Quick add of gc.collect

mvyboh committed:
Changed extraction of tar files
- changed the way tars are extracted
- bug fix related to multi-restart

mvyboh committed:
Removed double filter of raw data
- removed double filter when processing dBZ data

mvyboh committed:
complete_days list quick fix
- quick fix of improper complete_days list append method
